lib/checkother.cpp

```diff
@@ -1354,6 +1354,27 @@ static std::size_t estimateSize(const Type* type, const Settings* settings, cons
 
 static bool canBeConst(const Variable *var)
 {
+	{  // Ticket #8570
```

Comment:
I would like that the comment explains what this block tries to do. Like:

    // check initializer list. If variable is moved it can't be const.

Reply:
Thanks for the input, will change.